### DISCUSSION TITLE
I want to cancel the cancellation of the copilot.

### DISCUSSION THREAD

**Discussion by lacoroze** [community/community#33477] [Copilot Conversations] [answered]
My copilot expires in 4 days because I turned off the automatic payment. (Time of writing)
But I changed my mind and I want to make an automatic payment in 4 days or an annual payment.

However, no related options were found.

![image](https://user-images.githubusercontent.com/3162182/190984473-40572d53-effd-43db-9f2c-5adca11fd1d0.png)

I searched several times and expected that there would be a related option in the red circle of the attached image.

But there was nothing there.

Rather than changing the payment option after 4 days, I want to solve it now and use it comfortably.

**Comment by Akash1134** [comment]
Hi @lacoroze  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by lacoroze** [reply]
I've been a lot of trouble. I'm sorry.
Thank you for your answer!

**Comment by Sn3akySl0th** [comment]
I would like to cancel the copilot account and get a refund. I completely forgot that I was signed up for the free trial. Thank you.

**Comment by Akash1134** [comment, marked as answer]
Hi Everyone,

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.

**Comment by anon-board** [comment]
![image](https://github.com/user-attachments/assets/f18f92e3-83c2-4a05-b75b-d86961fe064c)

My cc is charged 2 days ago then i got this error. please just give a refund thank you.

**Comment by andreodendaal** [comment]
Same, I am trying to get a refund and get prompted to irrelevant options and i cannot open a ttcket

**Comment by pramalalsamarasekara-bit** [comment]
If this service is copilot, I want to unsubscribe from it. How do I do that?

**Comment by pramalalsamarasekara-bit** [comment]
Plase unsuscrbal my copilot  sarvis
